rasa/core/processor.py

```diff
@@ -732,6 +737,23 @@ async def parse_message(
 
         return parse_data
 
+    def _update_full_retrieval_intent(self, parse_data: Dict[Text, Any]) -> None:
```

Comment:
Could we add a processor unit test for this case please?

Reply:
Yep, just added it in the commit [acf376b](https://github.com/RasaHQ/rasa/pull/12998/commits/acf376b675d42acf8faaffff5564a2cad3e17ede)